packages/table/src/useLeafyGreenTable/useLeafyGreenTable.tsx

```diff
@@ -34,21 +35,24 @@ function useLeafyGreenTable<T extends LGRowData>({
   allowSelectAll = true,
   ...rest
 }: LeafyGreenTableOptions<T>): LeafyGreenTable<T> {
-  let hasSortableColumns = false;
+  const hasSortableColumns = useRef(false);
```

Comment:
why did you have to change this to use a ref?

Reply:
Looks like we're setting `hasSortableColums` inside the `columns` setter function. If this were a plain `let` variable it would be re-declared on every render, and would then be out of sync with the `columns` object